net.sf.eclipsecs.checkstyle/metadata/com/puppycrawl/tools/checkstyle/checks/coding/checkstyle-metadata.xml

```diff
@@ -398,6 +405,12 @@
             <message-key key="avoid.clone.method" />
         </rule-metadata>
 
+        <rule-metadata name="%NoEnumTrailingComma.name" internal-name="NoEnumTrailingComma" parent="TreeWalker">
+            <alternative-name internal-name="com.puppycrawl.tools.checkstyle.checks.coding.NoEnumTrailingCommaCheck" />
+            <description>%NoEnumTrailingComma.desc</description>
+            <message-key key="no.enum.trailing.comma" />
+        </rule-metadata>
+
```

Comment:
https://checkstyle.org/releasenotes.html#Release_8.29

> FallThrough module does not support the spelling "fall-through" by default.

Please update
https://github.com/checkstyle/eclipse-cs/blob/master/net.sf.eclipsecs.checkstyle/metadata/com/puppycrawl/tools/checkstyle/checks/coding/checkstyle-metadata.xml#L87

Hint: if release notes have something in "New" section .. it is almost for sure mean that eclipse plugin need be updated.

Reply:
![fallthrough](https://user-images.githubusercontent.com/4010811/73290727-ca777800-41b3-11ea-9f52-daaeeff93d8b.png)

Reply:
@Calixte , it is better for reviewer to resolve items, as ack that he is satisfied.
PR author can do comment "done" or disagrement comment.